public/docs/_examples/intro/dart/lib/hero_list_component_1.html

```diff
@@ -0,0 +1,7 @@
+<!-- #docregion -->
```

Comment:
The `#docregion`s are different in the ts version. I'm going to stop marking this, but it'd be great if this version had the same `#docregion`s as the ts version.